### PR TITLE
HTTPS Outcalls revisions

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-post.mdx [hunk 1]
```diff
@@ -224,7 +280,7 @@ actor {
     //     body : [Nat8];
     // };
 
-    //You need to decode that [Na8] array that is the body into readable text. 
+    /You need to decode that [Na8] array that is the body into readable text.
```
Comment (jennifertrin) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-post.mdx:283]:
```suggestion
    // You need to decode that [Na8] array that is the body into readable text. 
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-how-to-use.md [hunk 2]
```diff
@@ -1,33 +1,6 @@
...
-## Use cases
-There are many use cases for canister HTTPS outcalls, see the following list for some prominent examples.
-
-* One of the most important use cases is reading data from external HTTP APIs, e.g., pricing data used in DEXs or weather data used in decentralized insurance dApps.
-* IoT dApps need to obtain sensor data from traditional servers with which the sensors interact. In the future, the Internet Computer may even envision direct interactions of sensors.
-
-* Chat services sending push notifications about incoming messages to users.
-
-It is expected that the majority of HTTP calls to be `GET` calls for reading Web 2.0 data, but `POST` clearly also plays an important role for the interaction with external systems in order to be able to write data to Web 2.0 servers.
+This feature allows smart contracts to directly make calls to HTTP(S) servers external to the blockchain and use the response in the further processing of the smart contract, without the need of oracles.
```
Comment (jennifertrin) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-how-to-use.md:3]:
```suggestion
This feature allows smart contracts to directly make calls to HTTP(S) servers external to the blockchain and use the response in the further processing of the smart contract, without the need for oracles.
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-how-to-use.md [hunk 3]
```diff
@@ -36,51 +9,45 @@ It is expected that the majority of HTTP calls to be `GET` calls for reading Web
...
+-   `method`: Currently, only `GET`, `HEAD`, and `POST` are supported.
 
--   `headers`: list of HTTP request headers and their corresponding values.
+-   `headers`: List of HTTP request headers and their corresponding values.
 
--   `body`: optional; the content of the request's body.
+-   `body`: Optional; The content of the request's body.
 
--   `transform`: an optional function that transforms raw responses to sanitized responses, and a byte-encoded context that is provided to the function upon invocation, along with the response to be sanitized. If provided, the calling canister itself must export this function.
+-   `transform`: An optional function that transforms raw responses to sanitized responses, and a byte-encoded context that is provided to the function upon invocation, along with the response to be sanitized. If provided, the calling canister itself must export this function.
```
Comment (jennifertrin) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-how-to-use.md:36]: I suggest explaining what sanitized responses mean. Should we provide an example? Maybe giving an example of an API request that has a timestamp?

Comment (jessiemongeon1) [reply docs/developer-docs/integrations/https-outcalls/https-outcalls-how-to-use.md]: Thanks! I added a link to information about sanitization and provided an example.

**Review by dfx-json — APPROVED** [review]

**Pull request merged**